cabot/cabotapp/models.py

```diff
@@ -685,6 +685,12 @@ def description(self):
         help_text='Alert if build queued for more than this many minutes.',
     )
 
+    max_build_failures = models.PositiveIntegerField(
+        null=True,
+        blank=True,
+        help_text='Alert if more than this many consecutive failures (ignores other checks)'
```

Comment:
We can make this default to 0, then remove the regular logic for checking if the job succeeded because 0 consecutive failures already handles it? simplifies things a bit https://github.com/Affirm/cabot/pull/125/files#diff-65a212ee6aa85624abe3786856109453R753

otherwise right now the check will fail if the last build failed even if you set consecutive failures since it handles that logic after